kitegi/veg
# export veg targets and package config with version

steps to test:

1. build veg, say to `build`
2. install veg: `cmake --install build --prefix=temp/install`
3. import in a test project, say in `temp/CMakeLists.txt`: 
    1. `find_package(veg REQUIRED)`
    2. `cmake . -DCMAKE_PREFIX_PATH=./install`
4. check if configuration + generation succeeds, then attempt to use the library in code to find any headers that may be missing from `install` etc.
